### DISCUSSION TITLE
Dht11 sensor not working

### DISCUSSION THREAD

**Discussion by keshin88** [pixelit-project/PixelIt#208] [Q&A]
hello im running pixelit on a nodemcu v3 successfully i have connected the ldr sensor and have that working great, however no matter what pin i usei cannot get it to display humidity or temp. just says Not Installed, the nodemcu is new and i have used to brand new dht11 neither show installed. i have tried setting the pin to the one im using in the i2c and saved settings aswell as powering off and on and so far no success

**Comment by o0shojo0o** [comment]
Hi, it is because the DHT11 sensor is not supported. 
Only the DHT22 is supported from the DHT series. 

The DHT11 is unfortunately a very inaccurate sensor, so I would recommend you to install one of the supported sensors. 
Here are the currently supported:
[DHT22](https://pixelit-project.github.io/hardware.html#dht22-sensor-upgrade-temperature-humidity), [BME280](https://pixelit-project.github.io/hardware.html#bme280-sensor-upgrade-temperature-humidity-pressure), [BME680](https://pixelit-project.github.io/hardware.html#bme680-sensor-upgrade-temperature-humidity-pressure-gas)
